platform/ui/src/components/CinePlayer/CinePlayer.tsx

```diff
@@ -0,0 +1,146 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import debounce from 'lodash.debounce';
+import { Icon, Tooltip, InputRange } from '../';
+
+import './CinePlayer.css';
+import classNames from 'classnames';
+
+const CinePlayer: React.FC<{
+  className: string;
+  isPlaying: boolean;
+  minFrameRate: number;
+  maxFrameRate: number;
+  stepFrameRate: number;
+  frameRate: number;
+  onFrameRateChange: (value: number) => void;
+  onPlayPauseChange: (value: boolean) => void;
+  onClose: () => void;
+}> = ({
+  className,
+  isPlaying,
+  minFrameRate,
+  maxFrameRate,
+  stepFrameRate,
+  frameRate: defaultFrameRate,
+  onFrameRateChange,
+  onPlayPauseChange,
+  onClose,
+}) => {
+  const fpsButtonClassNames =
+    'cursor-pointer text-primary-active active:text-primary-light hover:bg-customblue-300 w-4 flex items-center justify-center';
+
+  const [frameRate, setFrameRate] = useState(defaultFrameRate);
+  const debouncedSetFrameRate = debounce(onFrameRateChange, 300);
+
+  const getPlayPauseIconName = () => (isPlaying ? 'icon-pause' : 'icon-play');
+
+  const handlePlayPauseClick = () => {
+    onPlayPauseChange(!isPlaying);
+  };
+
+  const handleFPSMoreClick = () => {
+    handleSetFrameRate(frameRate + 1);
+  };
+
+  const handleFPSLessClick = () => {
+    handleSetFrameRate(frameRate - 1);
+  };
```

Comment:
we can put these directly inside the jsx below, do we need to define another function for them? can't we just say `onClick: () => handleSet ...`